skbio/io/format/stockholm.py

```diff
@@ -117,8 +117,10 @@
 GF metadata is stored in the ``TabularMSA`` ``metadata`` dictionary.
 
 .. note:: When reading, duplicate GF feature names will have their values
-   concatenated in the order they appear in the file. When writing, each GF
-   feature will be placed on its own line, regardless of length.
+   concatenated in the order they appear in the file. Concatenation will
```

Comment:
make a similar note for GS metadata, and add a note below in the "tree handling" section about concatenation of multi-line trees